schema/mysql/schema.sql

```diff
@@ -1003,6 +1003,12 @@ CREATE TABLE stateful_set_owner (
   PRIMARY KEY (stateful_set_uuid, owner_uuid)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin;
 
+CREATE TABLE favorite (
+  resource_uuid binary(16) NOT NULL,
+  username varchar(255) NOT NULL,
```

Comment:
Should be `varchar(254) COLLATE utf8mb4_unicode_ci NOT NULL` to be consistent with the Icinga Web schema.